recipes/libarchive-c/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "libarchive-c" %}
```

Comment:
I'm not sure whether this is the best name. Sounds like a C library to me.
`python-libarchive-c` is the name of the repo, how about using that?